src/graph/client/graphClient.ts

```diff
@@ -51,6 +51,18 @@ window.onerror = (message) => {
   logToUI("ERROR: " + message);
 };
 
+window.addEventListener('resize', resizeCanvas, false);
+
+function resizeCanvas() {
```

Comment:
nit: canvas?

Reply:
I called it canvas as it was where the graph results are being drawn.  Did you have a name in mind?

Reply:
"Canvas" is a specific HTML element kind, I'm using an SVG instead, which is what is being resized